UI redesign and dark mode

ghislaineguerin opened the pull request:
<!-- If PR doesn't fully resolve the issue, replace 'Fixes' below with 'Related to'. -->
<!-- If there is no issue being resolved, open one before creating this pull request. -->
Fixes #4347 

<!-- Concisely describe what the pull request does. -->

This PR includes several UI enhancements focused primarily on improving the overall appearance of the application. Changes include:

- Updating color variables to align with new marketing guidelines.
- Adding support for dark mode, including relevant styles and variables.
- Implementing UI refinements to ensure design consistency.

### Design Consistency Improvements

- Removed unnecessary layout variations across different pages (homepage, database page, schema page).
- Introduced adaptive layouts based on content requirements:
  - **Homepage:** Uses stacked cards optimized to display detailed database connection and for showing few items.
  - **Database Page:** Employs a grid layout of schema cards for ease of navigation.
  - **Schema Overview:** Utilizes a list format for efficient visual scanning of tables, facilitating future enhancements like sorting based on metadata (e.g., recent updates, favorites).

### De-cluttering Table and Exploration Views

- Simplified layouts by reducing excessive borders that previously divided the screen into disconnected sections, reducing visual noise.
- Made grids, inspectors, and other panels more modular, which supports easier integration of additional elements in the future without complicating the UI.

### Database name simplification

- I've added a small change to only show the original database name if there's a display name selected. 

### Variables 

- I've updated variables but still a work in progress to remove redundant styles or overly general ones.

## Checklist
<!-- Replace  the [ ] with [x] to check the boxes. --> 
- [ ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [ ] My pull request targets the `develop` branch of the repository
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin
<!-- You must read and understand the following attestation. -->

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

zackkrida commented:
@ghislaineguerin since you've been out for a bit I thought a preliminary review might be useful. Here are all of the things I'm seeing:

### Global

- Dropdown button functionality is broken:
  - The edit buttons for schema and table cards are not triggering the modals properly
  - The "+ New Table" => "From Scratch" button action isn't opening a modal 
  - The filter, sort, and group buttons aren't actually applying filters
  - _This is the only critical issue I see_

### Import flow

- The inputs to edit column names are cutoff at the top:
  ![image](https://github.com/user-attachments/assets/adcc5677-52c8-4b40-a418-b50af868ceb3)


## Dark Mode

###  Global

- Odd double focus ring (white inner ring) on header "feedback" button
  ![image](https://github.com/user-attachments/assets/df9b2351-416c-4a3f-a012-fdba0d9bc112)
- I'm not sure the purple text for input 

### Database list page

- Database background color when databases require an upgrade `.db-card-needs-update { background: var(--gray-100); }`
  ![image](https://github.com/user-attachments/assets/0f1d9262-ebf3-4d83-b88f-80cf18ef84e2)

### Schema list page

- In "Database Setings", the Roles and Collaborator tables have hardcoded white backgrounds (see the corners):
  ![image](https://github.com/user-attachments/assets/bdcefb19-ec0f-47ca-816f-aa813c661348)

### Table list page

- The outer ~10px of the table boxes aren't clickable (see that the pointer isn't a screenshot):  
  ![image](https://github.com/user-attachments/assets/6072c830-22eb-42cd-9d3e-d0ad356daf5e)
 

### `/profile/`

- Incorrect background color (you know this one already)
  ![image](https://github.com/user-attachments/assets/cdbe9f9d-499a-4a89-bc5d-b8e136f3a18b)

  
### Data explorer

- Selected column heading cell color is incorrect:
  ![image](https://github.com/user-attachments/assets/95b3f910-8f00-468e-baf6-b63ba79f0bdf)
- Footer toolbar background is incorrect:
  ![image](https://github.com/user-attachments/assets/a45cb6dd-926c-4eb6-9ab3-9bf0ead41012)
- Tag background color is incorrect:
  ![image](https://github.com/user-attachments/assets/3c67e17c-6c2b-45e9-83d3-ef19b2a815a7)

### Single table view

- "New record reposition" notice and "All changes saved" toast background colors are incorrect:
  ![image](https://github.com/user-attachments/assets/1ed7b196-005d-4c6f-a565-30e7d9a3faab)
- The same color is used for the record selector focused item and is not ideal:
  ![image](https://github.com/user-attachments/assets/78eb22c0-e7e7-4cb6-8e3c-9f0423793d85)
- Error messages are illegible:  
  ![image](https://github.com/user-attachments/assets/b7424af7-c320-40ac-abd4-c547526ebbdb)
- The "selected" inspector background might be too subtle.

  | Selected | Deselected |
  |--------|--------|
  | ![image](https://github.com/user-attachments/assets/ddc57329-0517-43ea-ae67-be165f775008)| ![image](https://github.com/user-attachments/assets/c785f093-1916-4e7b-9ad1-e6bb063fd260) |

ghislaineguerin commented:
@zackkrida I've fixed the bugs and added some additional changes to increase contrast. There's an error I cannot fix. Other than that I think it's ready for review.

zackkrida commented:
@ghislaineguerin I really like the changes, particularly those on the table view! The only visual bug I still see is missing padding on small breakpoints:

![image](https://github.com/user-attachments/assets/6ced4710-5ceb-42d6-b920-ff9e4bd5323b)
![image](https://github.com/user-attachments/assets/6309fed4-14dc-4009-b3fb-56f212d9bf39)

Can you fix that? After that I think you should mark the PR as ready for review from Sean and I, where we can focus on the design itself along with any new bugs we identify. Thanks!

seancolsen reviewed (changes requested):
## Notes

- I'm mostly reviewing Mathesar in _light mode_ — so that I can focus on finding regressions. But I'm also reporting issues with dark mode that seem serious enough to hinder usage.

## Overall impressions

- I see lots of stylistic improvements and nice-looking changes here! Nice work!

## Changes I pushed

- In a2515c8509811656ca5efc084b9d661c584b6ff2, I changed some HTML `title` attributes to use our `Tooltip` component instead. This is pretty minor. But I've been moving towards this pattern more recently because we can do more with tooltips (e.g. rich text, more control over position, etc), and I think it's nice to try making these tooltips consistent across the UI as much as possible.

## High Priority Problems

### 1

On multiple pages, there is a layout problem causing content overflow on narrow viewports.

![image](https://github.com/user-attachments/assets/2f94e918-8462-4614-b789-14a61c019546)

![image](https://github.com/user-attachments/assets/2e5e4b6f-11c4-4070-a81d-cc6ad5a33aea)

### 2

It seems that all pages under "Administration" have a problem where it is not possible to vertically scroll the page. This leaves content off screen that cannot be accessed.

![image](https://github.com/user-attachments/assets/fba2ab58-f79f-4012-9e3b-000e71fe6794)

![image](https://github.com/user-attachments/assets/95a0455c-1d59-49d3-af12-b0b124a919c1)

### 3

The contrast here too low to actually be usable

![image](https://github.com/user-attachments/assets/dc8852c1-fe24-4e4d-b308-7a82b5ed5243)


## Smaller usability issues

### 4

On the home page, I think "Resources" should display below "Databases" on narrow viewports

![image](https://github.com/user-attachments/assets/82361ac4-6274-44a2-8d58-629bf8d6518f)

### 5

When a database needs to be upgraded, this chevron should not appear when hovering the card:

![image](https://github.com/user-attachments/assets/3f7367db-8718-449f-91b3-8c299ad68d7f)

### 6

This contrast is too low for me to read the question mark:

![image](https://github.com/user-attachments/assets/e41c9cbf-ca6d-4b62-afcf-9e99c45e6a64)

### 7

This seems like the wrong color:

![image](https://github.com/user-attachments/assets/4320667f-41e5-4dba-bac9-659b91e1e74d)

### 8

I really don't like the dual scroll bars here. I would much rather scroll the page to show tables instead of scrolling the table area. This is true on wide viewports too, although the problem is worse on narrow viewports.

![image](https://github.com/user-attachments/assets/13a073e3-9772-4901-9244-82ffff592918)

### 9

There is a small horizontal overflow on the table widget causing horizontal scrolling even when all content is visible

![image](https://github.com/user-attachments/assets/5f7801c0-c951-421a-9a18-5edc602e0257)

### 10

Low contrast here

![image](https://github.com/user-attachments/assets/3c1b4fe0-d0ce-4cd9-a204-ea2ae3484cfe)

![image](https://github.com/user-attachments/assets/11a41c7e-70b9-4bbd-a040-b5aaff3ce60c)

### 11

Missing padding here

![image](https://github.com/user-attachments/assets/cc423b08-b71b-48d5-8390-89f0198c98bf)

### 12

Missing breadcrumb separators here:

![image](https://github.com/user-attachments/assets/30a9c5bf-81ba-410f-8ba5-6899402cdc4c)

### 13

The hierarchical structure of this navigation menu is harder to discern, given the new styling. At a glance it appears to me that there are five menu items, two of which are disabled.

![image](https://github.com/user-attachments/assets/00a6d0f7-76bc-4d5e-bddc-149fc028f297)

### 14

When a cell is selected, its content moves down a little bit. This should not happen. The cell content should not move around when the cell is selected.

[Screencast_20250418_112430.webm](https://github.com/user-attachments/assets/12140a70-1611-468e-b5b0-d4f06942f08e)

### 15

In dark mode the cell selection background seems too subtle for dark mode users to be able to easily determine which cells are selected.

![image](https://github.com/user-attachments/assets/bed4493f-b2e5-4b8d-9b14-243653a7d054)

### 16

Contrast too low here

![image](https://github.com/user-attachments/assets/d9c4610e-d2ac-447f-8f21-86f2f7ee8979)

![image](https://github.com/user-attachments/assets/6dbf5e76-0fcf-4e06-b8bd-e27287ee6923)

### 17

Something is funky with this Linked Record input. I think it might have a fixed height set that needs to be adjusted to account for the new height of the other elements:

![image](https://github.com/user-attachments/assets/a461aa8c-bcac-4d98-ad27-c58e2b26b747)

![image](https://github.com/user-attachments/assets/318b8a96-6079-41e3-8e7f-01331b35cdab)

### 18

Previously, these trash can icons had a background. Without the background, it looks weird when dragging to modify sorting precedence.

![image](https://github.com/user-attachments/assets/05ebf530-95fe-4bce-8417-bb29d2a2dc9a)

### 19

Contrast a bit low here

![image](https://github.com/user-attachments/assets/14fd334a-eeb3-47ab-9390-18c5158a6cd8)

### 20

Within the multi-select, the color of selected items is now red, whereas previously it was blue. Red seems inappropriate here. It gives me the impression that maybe there was an error with the column I chose.

![image](https://github.com/user-attachments/assets/db49d2c4-a63f-4b99-abe6-0a75ed8bf44c)

### 21

Low contrast

![image](https://github.com/user-attachments/assets/80ec98ac-eab5-44ea-86c0-2b443b6511a4)

### 22

Schema icon missing here

![image](https://github.com/user-attachments/assets/f4c23243-9907-4542-8412-61cfa3097b65)

### 23

Array cell pill background is quite low contrast in the cell inspector. Perhaps we use a white background for the entire cell content within this context so that it matches the background used in the table context?

![image](https://github.com/user-attachments/assets/aaa77c89-da49-4430-a40a-2e75d360e482)

### 24

When a record has no related records, it would be nice to keep the background consistent throughout the entire hight of the page instead of having this white block at the bottom.

![image](https://github.com/user-attachments/assets/cade5773-baf6-4c70-8657-f223fc9f63dd)

### 25

When a modal appears in dark mode it seems difficult to distinguish a modal from the application content behind it. I don't use dark mode though, so perhaps this behavior is typical of applications in dark mode? I'm not sure how much of a problem it will be. I do notice that the overlay _does_ seem to make white text below the modal slightly less white. Perhaps we could increase that effect? I'm not sure.

![image](https://github.com/user-attachments/assets/42d8ff65-686d-4ff7-9bc7-599ea831887e)


## Aesthetic opinions

### 26

Personally I don't like the increased padding on buttons. I would expect that most of our users would value information density in an application like Mathesar. But I could be wrong.

### 27

I find the cell-level hover background to be distracting here. I don't see what purpose it serves. I suggest removing it.

![image](https://github.com/user-attachments/assets/0908193d-49b3-44c3-8ffd-01389bb1b9c9)

### 28

The group header row lost some of its padding. I think that padding was helpful in visually distinguishing the header from the other rows. I'd like to see a bit more padding on the bottom of the group header row. The top is fine as-is.

![image](https://github.com/user-attachments/assets/df389dbe-2f77-4e86-97df-7f5097618f81)

### 29

I'd like to see some hover styling when navigating to tables from the tables list

![image](https://github.com/user-attachments/assets/0e6a3b2b-04f4-451d-b484-f046a1ab2919)

### 30

I don't really like how prominent these dividing lines appear. I'd like to see less contrast here so that it's easier to visually parse without getting lost in the lines. I'd also be okay with removing the lines entirely.

![image](https://github.com/user-attachments/assets/f0c0a4bc-4b2f-43dd-8441-7e7ff1ab8b88)

seancolsen reviewed (commented):
Ok I got a chance to review the code now too. I made some comments inline. And I also have the following points of feedback:

- We have a [code standard](https://github.com/mathesar-foundation/mathesar/blob/develop/mathesar_ui/STANDARDS.md#component-spacing-and-layout) that says:

    > The component's root element should not set any margin.

    I'm okay with the margin you've set on the root element of `ExplorationItem.svelte`. It's clever what you're doing there, and I think the tradeoff is worth it. Similar for `LogoAndNameWithLink.svelte`. Not a big deal there. I don't think we need to be super strict about this code standard.

    But for the following other components, you seem to have added new margin to the root element, and I think these might warrant a closer look.

    - `Sheet.svelte`
    - `ExplorationInspector.svelte`
    - `TableInspector.svelte`
    - `InputSidebar.svelte`
    - `AppSecondaryHeader.svelte`

    I dug into `Sheet.svelte` as an example. The root margin there seems to be for the purpose of separating the sheet from its sidebars on the table page and data explorer. But it also seems to be the cause of problem 9 that I mentioned in my previous review. And the margin is not necessary in the one other context where Sheet.svelte is used — the import preview page. I would consider implementing that spacing in the `WithPanel` component instead (in a configurable manner). That way we could fix a small usability problem that I noticed while testing this — when I resize a panel, I expect that I can click in the middle of the space that divides the panels, but that doesn't work.

    There may be other similar considerations with the other components that have new root margin set. And also maybe not. If you look at them and decide to keep the root margin, that's okay with me, as long as you've given it some thought. I've just found it to be much cleaner and easier to compose components when they don't add any extra space around themselves. Try to add that space within the scope of the consuming component if possible.

- I notice you've added `z-index` CSS in `Tutorial.scss`, `SheetHeader.svelte`, and `BreadcrumbSelectorRow.svelte`. As far as I can tell, none of these instances conform to our [code standards](https://github.com/mathesar-foundation/mathesar/blob/develop/mathesar_ui/STANDARDS.md#component-spacing-and-layout) for z-index. I don't raise this just to be pedantic. Tracing down z-index bugs has been a real headache for me with Mathesar in the past, due to the complexity of our UI. So in this case I haven't spent the time to trace down any specific bugs arising from this new z-index CSS. But I would just like it to conform to the standards so as to avoid any bugs going forwards.

    Basically: any time I see a number after `z-index`, I want to also see `isolation: isolate` set on an ancestor element _within the same component_. By itself, `position: relative;` does not establish a new stacking context, and this is how we've ended up with z-index bugs in the past.

    At first glance, I think you ought to be able to use `isolation: isolate` in `Tutorial.scss` and `BreadcrumbSelectorRow.svelte` because in those cases you appear to be setting `z-index` on child elements within the component.

    However in `SheetHeader.svelte` you appear to be setting `z-index` on the root element within the component. So this one will need different treatment. I see that in `Sheet.svelte` there is a big block of CSS to handle z-index values across various components within the "sheet" system. So I'm guessing there may be an appropriate z-index value CSS variable to choose from in that list. But you'll need to give this some thought, in conjunction with the problem you were originally trying to solve when adding the z-index code.

- I notice you've added `line-height: 1;` in `Button.scss` and `Menu.scss`. I've traced down usage of this CSS into the application and it doesn't seem to actually cause any issues in Mathesar _right now_. But it can cause issues [like this](https://github.com/mathesar-foundation/mathesar/issues/4332) _later_ if it lurks in low-level code that gets reused or modified. I'm guessing you adjusted the line-height to modify some spacing. Can you adjust margin or padding instead?

    Note that there are a few other places we have `line-height: 1;` in the codebase. Ideally I'd like to get rid of all of them. I'm just commenting about these two instances of it now to try to nip it in the bud since it's code you have just worked on recently.

- You've added `:global` in `Button.scss` and `Tutorial.scss`. You don't need to use `:global` in SCSS files since everything is global there. It's not hurting anything. But can you remove it so that it doesn't confuse others in the future?

- I see 9 occurrences of `transition: all` in the codebase, and it seems that 7 of them were added in this PR. In general I'd like to avoid `transition: all` because it can lead to inadvertent transitioning if we later choose to dynamically modify other properties interactively. What do you think about changing these to explicitly specify the properties being transitioned? The 7 occurrences that you've added seem to be:

    ```
    mathesar_ui/src/component-library/button/Button.scss
    +  transition: all 0.15s ease;
    +    transition: all 0.2s ease-in-out;

    mathesar_ui/src/components/breadcrumb/BreadcrumbSelector.svelte
    +    transition: all 0.2s ease-in-out;

    mathesar_ui/src/pages/database/schemas/SchemaRow.svelte
    +    transition: all 0.2s ease-in-out;

    mathesar_ui/src/pages/home/database-card/DatabaseCard.svelte
    +    transition: all 0.2s ease-in-out;

    mathesar_ui/src/pages/schema/ExplorationItem.svelte
    +    transition: all 0.2s ease;

    mathesar_ui/src/systems/table-view/table-inspector/table/links/LinkCard.svelte
    +    transition: all 0.2s ease;
    ```

    Found with:

    ```sh
    q="transition: all"
    for f in $(git diff develop --name-only --diff-filter=AM); do
      lines=$(git diff develop -- "$f" --unified=0 | grep '^+[^+]' | grep "$q")
      if [ -n "$lines" ]; then
        echo "$f"
        echo "$lines"
        echo ""
      fi
    done 
    ```

- There are some unused variables and imports after your changes. Please run eslint, look at the warnings, and remove unused imports on files you've changed. This helps keep our eslint warnings low.

ghislaineguerin commented on mathesar_ui/src/AppContext.svelte:
```diff
@@ -80,7 +83,7 @@
 </script>
 
-<svelte:body on:copy={handleCopy} on:paste={handlePaste} />
+<svelte:body on:copy={handleCopy} on:paste={handlePaste} class="test-1-2-3" />
```
debug code removed in 59ac9e5e100fd0cf60338a8002abb83d296beb39

ghislaineguerin commented on mathesar_ui/src/components/SchemaName.svelte:
```diff
@@ -5,16 +5,13 @@
 </script>
 
 <span class="schema-name" class:is-locked={isLocked}>
-  <NameWithIcon icon={iconSchema} {iconHasBox}>
```
Yes, that's what I was trying to do. I remove the component from schema row now.

ghislaineguerin commented on mathesar_ui/src/pages/database/schemas/SchemaRow.svelte:
```diff
@@ -60,20 +66,32 @@
+    </div>
+  </div>
 
+  <!--
```
I've removed it now.

seancolsen reviewed (changes requested):
Thanks for the adjustments, @ghislaineguerin. I think we're almost there. Here's what I've noticed now. Some of these are from before, and some are new. I've noted ones that are small enough to ignore if you prefer.

## Code issues

- It doesn't seem that you've addressed the z-index issue in `SheetHeader.svelte` or `BreadcrumbSelectorRow.svelte`.

## Previous UI issues

- It doesn't seem that you've addressed issue `14` in my original review.

## New UI issues

### 31

The Database page is not very good at handling long schema names.

![image](https://github.com/user-attachments/assets/1111dee9-8d73-4e51-be66-14cc44589ec7)

### 32

I see you've fixed the hover behavior for these cells, but it's still weird to me that clicking on them gives them background. Can we remove that?

![Image](https://github.com/user-attachments/assets/2591791e-a141-4abf-81ba-a3e4c8807a43)

### 33

I'd like to see a little space at the very bottom of pages after scrolling:

![Image](https://github.com/user-attachments/assets/9468e7ed-2829-4bec-a2da-e11ff6e0780c)

### 34

The horizontal scrolling of table widgets is still a bit weird. There is some extra space on the left that doesn't seem like it should be there. And it seems to me like the border radius and border ought to be there even when the some content is scrolled out of view horizontally. This is a pretty small issue though. I'd be okay leaving it.

![Image](https://github.com/user-attachments/assets/ea6ae6b0-9bf6-4972-a119-ed3f82702923)

### 34

The table loading state in dark mode is not very dark. This is pretty minor, and I'd be okay leaving it if you prefer.

![Image](https://github.com/user-attachments/assets/0ebf532a-0c48-4e78-9920-8ce576314702)

seancolsen reviewed (changes requested):
## Code issues

- I'm still seeing the z-index issue in `BreadcrumbSelectorRow.svelte`. Since this is the third time I've mentioned it, I'm wondering if there might be a reason you're intentionally not addressing. If you feel that it doesn't need to be fixed, let me know. Or if you are not sure how to proceed, let me know. I'd be happy to chat about it!

## UI issues

### 35

This alignment seems off

![image](https://github.com/user-attachments/assets/58f7d45a-cec4-4418-8d1b-15110bbd3b35)

### 36

Your fix to `14` seems to have introduced this subtle regression to the appearance of the active cell outline. The outline now seems to be a bit too big vertically.

![image](https://github.com/user-attachments/assets/bb461aed-67c6-4923-8692-2284f8779d24)

### 37

I notice that the text now moves up when editing a cell.

![image](https://github.com/user-attachments/assets/ea666fdf-6764-4ff8-bb08-26333d4a0a91)

At the very least, this is a minor annoyance, making it harder for the user to retain context as they edit. For example, I think it's common to double-click a cell and then click again to place the cursor at exactly the spot where you double-clicked. That is no longer possible now that the text moves up.

But what's much worse is that editing cells in the first row becomes really hard:

![image](https://github.com/user-attachments/assets/ae9b00e4-8d1d-4e90-bde9-a83425211d26)

ghislaineguerin commented:
@seancolsen For some reason I've missed the z-index, sorry about that it was not intentionally missed.

ghislaineguerin commented:
@seancolsen I've managed to fix the alignment issues but I seem to be missing something with the cell wrapper (the blue selection area) and can't get it to work. Would you mind taking a look at it or maybe we can jump on a call and figure it out?

seancolsen commented:
Sure @ghislaineguerin, I'll look into the cell wrapper issues.

In the mean time, I'd like to point out another issue I noticed:

### 38

When the active cell is focused, it's easy to distinguish it from non-active cells which are also selected. This is good:

![image](https://github.com/user-attachments/assets/d2f05de0-881d-4d52-b20b-9a101d7a98f3)

But when the active cell is _not_ focused, it becomes difficult to distinguish it from the other selected cells. This is not good:

![image](https://github.com/user-attachments/assets/9f049fcd-bf93-490e-aa83-368ed6f64306)

I think the border needs to be darker here.

Can you address this one?

seancolsen commented:
@ghislaineguerin

- I see you fixed `35`. Thank you.
- I pushed 91e05e4cb5a8143a76255265c7e195471a14ca6f which fixes `36` and `37`.
- I'll let you fix `38` so that you can choose the color. As far as I'm concerned the outline (it's technically implemented as a shadow) just needs to be dark enough to have more contrast. See the `box-shadow` declarations in `CellWrapper.svelte`.

ghislaineguerin commented:
@seancolsen Thanks for your help! I've pushed an update for the cell wrapper color.
